spring-cloud-dataflow-shell-core/src/test/java/org/springframework/cloud/dataflow/shell/command/TaskScheduleCommandTemplate.java

```diff
@@ -67,11 +71,24 @@ public void create(String name, String definition, String expression, String pro
 				"task schedule create --name \"%s\" --definitionName \"%s\" --expression \"%s\" --properties \"%s\" --arguments \"%s\"",
 				name, definition, expression, properties, args);
 		CommandResult cr = dataFlowShell.executeCommand(wholeCommand);
-		verify(schedule).schedule(name, definition, Collections.singletonMap("scheduler.cron.expression", "* * * * *"),
+		verify(schedule).schedule(name, definition, Collections.singletonMap("scheduler.cron.expression", expression),
 				Collections.emptyList(), null);
 		assertEquals("Created schedule 'schedName'", cr.getResult());
 	}
 
+	public void createWithPropertiesFile(String name, String definition, String expression, String propertiesFile, String args) throws IOException {
+		String wholeCommand = String.format(
+				"task schedule create --name \"%s\" --definitionName \"%s\" --expression \"%s\" --propertiesFile \"%s\" --arguments \"%s\"",
+				name, definition, expression, propertiesFile, args);
+		CommandResult cr = dataFlowShell.executeCommand(wholeCommand);
+
+		Map<String, String> expectedProperties = DeploymentPropertiesUtils.parseDeploymentProperties("", new File(propertiesFile), 1);
+		expectedProperties.put("scheduler.cron.expression", expression);
+
+		verify(schedule).schedule(name, definition, expectedProperties, Collections.emptyList(), null);
+		assertEquals("Created schedule 'schedName'", cr.getResult());
```

Comment:
Probably want to use the `name` parameter that was passed into the method to do this assertion instead of hardcoding `schedName`.